utils/src/ooinstall/ansible_plugins/facts_callback.py

```diff
@@ -27,9 +27,19 @@ def on_any(self, *args, **kwargs):
     def runner_on_failed(self, host, res, ignore_errors=False):
         pass
 
```

Comment:
Well, it is only blocking people who are using ansible 2.1 and/or ansible on Fedora 23, of whom I seem to be the only one . . .